cmake/modules/SetROOTVersion.cmake

```diff
@@ -82,7 +82,9 @@ function(SET_ROOT_VERSION)
     if (${GIT_DESCRIBE_ALL} MATCHES "^tags/")
       string(REGEX REPLACE "^tags/" "" GIT_DESCRIBE_ALWAYS ${GIT_DESCRIBE_ALL})
     else()
-      message(WARNING "GIT_DESCRIBE_ALL is `${GIT_DESCRIBE_ALL}` with a format unexpected for tags. Not setting the commit.")
+      set(ROOT_GIT_VERSION "${ROOT_MAJOR_VERSION}-${ROOT_MINOR_VERSION}-${ROOT_PATCH_VERSION}")
```

Comment:
Should we check that those variables are set before using them? (and if they are not still issue the warning?)

Reply:
Well, the complain from Atlas is about the warning, not about the variables not being set...

Reply:
And I just found more elegant to artificially set the variables using `RVersion.h` than letting them empty...

Reply:
That sounds good.  Do I understand correctly that `ROOT_MAJOR_VERSION`, `ROOT_MINOR_VERSION` and `ROOT_PATCH_VERSION` are always guaranteed to be set and we will never ends with `ROOT_GIT_VERSION == "--"`?

Reply:
They are taken from `${CMAKE_SOURCE_DIR}/core/foundation/inc/ROOT/RVersion.hxx`. So if they are not guaranteed to be set then we are in big trouble I think...

Reply:
Perfect.  Thanks for the clarification.